client/src/i18n/en.json

```diff
@@ -271,10 +271,11 @@
       "VIEW_ERRORS"           : "View Errors"
     },
     "DIALOGS": {
+      "CONFIRM_ACTION"         : "Do you confirm this action ?",
       "CONFIRM_DELETE"         : "You are about to delete a record.  This action is permanent and cannot be done.  Click \"Confirm\" to accept or \"Cancel\" to halt this action.",
-      "CANNOT_UNDONE_DELETION" : "This action CANNOT BE REVERSED, the deletion will be permanent. You must be sure that you want to do this action",
-      "NO_CORRESPONDANCY"      : "No %ELEMENT% have the name ",
-      "PLEASE_TYPE_TEXT"       : "Please type in the %ELEMENT% name to confirm",
+      "CANNOT_UNDONE_ACTION"   : "This action CANNOT BE REVERSED, this action will be permanent. You must be sure that you want to do this action",
+      "NO_CORRESPONDANCY"      : "No correspondancy found for ",
+      "PLEASE_TYPE_TEXT"       : "Please type %ELEMENT% to confirm",
```

Comment:
For future reference, angular-translate provides variable substitution so that you do not have to do this manually.  [Documentation](https://angular-translate.github.io/docs/#/api/pascalprecht.translate.filter:translate).  [An interesting PR related to this](https://github.com/angular-translate/angular-translate/issues/188).

Since we've been using this style long term, I don't mind merging this PR.  But if possible, it would be nice to move to `angular-translate`'s way of putting parameters into our translations.  I made an issue about it a long time ago: https://github.com/IMA-WorldHealth/bhima-2.X/issues/408.